bin/clover/src/pipeline-steps/generateAssetFuncs.ts

```diff
@@ -81,6 +81,28 @@ function generateAssetCodeFromVariantSpec(variant: SchemaVariantSpec): string {
     adds += propAdds;
   }
 
+  // Code for Secret Props
+
+
+  {
+    if (variant.secrets.kind !== "object") {
+      console.log(
```

Comment:
I think we should error here as we'll generate a bad asset func without this, right?